pkg/project/v2/project.go

```diff
@@ -62,10 +68,38 @@ func (p Project) HasDependencyOn(environment string, project Project) bool {
 	return false
 }
 
-func (p Project) String() string {
+func (p *Project) String() string {
 	if p.GroupId != "" {
 		return fmt.Sprintf("%s [group: %s]", p.Id, p.GroupId)
 	}
 
 	return p.Id
 }
+
+func (p *Project) ForEveryConfigDo(actions ...ActionOverConfig) {
+	for _, cpt := range p.Configs {
+		for _, cs := range cpt {
+			for _, c := range cs {
+				for _, f := range actions {
+					f(c)
+				}
+			}
+		}
+	}
+}
+
+func (p *Project) GetConfigurations(filter ...func(c config.Config) bool) []config.Config {
```

Comment:
I will be removing this method (it is unused anyway).
Getting all configurations in a project does generally not make sense, as they are duplicated per environment (with possible overwrites).

So for a given project of a single config, in a manifest defining three environments, this method would return a slice containing that config three times. 

Access to a projects configurations should always be for a specific environment, which can be done via the Env -> ConfigsPerType map.